ush/config_defaults.yaml

```
@@ -2962,6 +2976,37 @@ task_aqm_lbcs:
       tasks_per_node: 24
       walltime: 00:30:00
 
+# ------------------------------------
+# SRW-AQM MELODIES-MONET Configuration
+# ------------------------------------
+task_mm_prep:
+  execution:
+    batchargs:
+      nodes: 1
+      tasks_per_node: 1
+      walltime: 00:05:00
+  # Output directory for MELODIES MONET-generated model evaluation plots and statistics.
+  MM_OUTPUT_DIR: null
+  # Evaluation packages to initialize and run.
+  MM_EVAL_PACKAGES: [chem]
+  # If set to another SRW experiment path, MELODIES MONET will generate "scorecards" for model inter-
+  # comparison.
+  MM_BASE_MODEL_EXPT_DIR: null
+  # Path, optionally with wildcards, selecting the AirNow observation files used by MELODIES MONET.
+  # For example: /staged/obs/Observations/AirNow/AirNow_2023*.nc
+  MM_OBS_AIRNOW_FN_TEMPLATE: null
+task_mm_run:
+  execution:
+    batchargs:
+      nodes: 1
+      tasks_per_node: 100
```

Setting `tasks_per_node` to a value over 40 (Hera and Orion) or 80 (Hercules) will cause the `mm_run` tasks to fail to submit.  The machines that have a `tasks_per_node` count of over 100 are Derecho, Gaea C6, and Ursa.  Setting `tasks_per_node` to `platform.NCORES_PER_NODE` should get around this issue.  If we only want MM tasks to run on Derecho, Gaea C6, and Ursa, then the documentation should be updated to include this.

```suggestion
      tasks_per_node: !int '{{ platform.NCORES_PER_NODE }}'
```

---

Done! Thx for pointing this out.